examples/data-sources/integration/data-source.tf

```diff
@@ -0,0 +1,12 @@
+data "tanzu-mission-control_cluster" "default" {
+  management_cluster_name = "attached"       # Default: attached
+  provisioner_name        = "attached"       # Default: attached
+  name                    = "terraform-test" # Required
```

Comment:
Can you please check the examples and correct the keys. 
I believe this should be integration_name